### PR TITLE
WIP: Testing homepage content for authenticated user

### LINKED ISSUE #103: Test all stations are visible to the logged in user.
As a developer, I want to make sure that successfully logged in user can see all the stations on the homepage so (s)he can start ordering process.

GIVEN - user signed in
WHEN - homepage is displayed
THEN - all stations (that are in database) are present on the homepage
AND - every station can be 'clicked on'

### PR DESCRIPTION
**Pull request by lemmetry**
Testing that for the authenticated user all stations are shown on the `homepage` and each station has a link to respective `stations_dashboard_page`.

TODO: Refactor class/method names.

### CODE REVIEW

File: fleet/models.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 
 class Station(models.Model):
-    station_id = models.CharField(max_length=2)
```
Comment (agibalov) [inline fleet/models.py:8]: What we discussed is that you should only use `station_id` for display purposes (but not for references), and you should use `pk` for references (but not for display purposes). The way it is right now is, you have zero control over what becomes a station 1, station 2, etc, and you can make a station 10 without first making 1, 2, 3, ..., 9.

File: fleet/models.py [hunk 2]
```diff
@@ -22,17 +21,15 @@ def format_phone_number(self):
         return
 
     def get_name(self):
-        name = 'Station ' + self.station_id
-        return name
+        return f'Station {self.pk}'
```
Comment (agibalov) [inline fleet/models.py:24]: So you can't have station 1000 without having station 1, station 2, ...., station 999. Right?

File: fleet/models.py [hunk 3]
```diff
@@ -22,17 +21,15 @@ def format_phone_number(self):
...
 
     def get_logo(self):
         if self.logo:
             return self.logo.url
         return static('fleet/logo_jcc.png')
 
     def get_path(self):
```
Comment (agibalov) [inline fleet/models.py:31]: Where/how do you use this?

**Review by agibalov — CHANGES_REQUESTED** [review]

File: inventory/views.py [hunk 4]
```diff
@@ -49,8 +48,8 @@ def make_station_order(request, order_pk):
     station_order = get_object_or_404(StationOrder, pk=order_pk)
     station = station_order.station
     station_name = station.get_name()
-    station_id = station.station_id
-    station_fleet = Vehicle.objects.filter(station=station_id)
+    station_pk = station.pk
```
Comment (agibalov) [inline inventory/views.py:51]: Do you believe that `station_pk` is more descriptive, than `station.pk`?

File: fleet/models.py (hunk 3 above)
Comment (lemmetry) [reply fleet/models.py:31]: https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L27
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L50
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L78
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L184
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L241

File: inventory/views.py (hunk 4 above)
Comment (lemmetry) [reply inventory/views.py:51]: If I need to use the value of `station.pk` multiple times within the function - I'd rather assign a variable to it than calling the `station` object.

**Review by agibalov — APPROVED** [review]

**Pull request merged**